### PR TITLE
Bump version

### PR DESCRIPTION
**Pull request by tomrosenback**
Bumping version to verify auto-update. Merge only after "last" outstanding commits / PRs are merged in both repos.

### CODE REVIEW

**Comment by Jalle19** [comment]
Fine by me, feel free to merge.

**Comment by Jalle19** [comment]
@tomrosenback IMO let's keep the addon version separate from the application version, so 1.0.1 is perfectly fine!

**Comment by Jalle19** [comment]
Seems to require a rebase, after that feel free to merge!

**Comment by tomrosenback** [comment]
After sleeping on the versioning, I think I would prefer that the addon version follows the bridge version, this way it is clearer which version of the bridge is released as an addon also. Ok with that?

**Comment by Jalle19** [comment]
Sure, I'm okay with that.

**Comment by tomrosenback** [comment]
As the slug has changed, this version will appear as a separate addon. So uninstall previous version and then install this one.

File: eda-modbus-bridge/CHANGELOG.md [hunk 1]
```diff
@@ -1,11 +1,17 @@
...
+[eda-modbus-bridge CHANGELOG](https://github.com/Jalle19/eda-modbus-bridge/blob/master/CHANGELOG.md) for changes to the
 application itself.
 
 The version number here doesn't necessarily reflect `eda-modbus-bridge`'s version numbering.
 
 ## 1.0.0
 
 Initial release as a proper Home Assistant add-on!
+
+## 1.0.1
```
Comment (Jalle19) [inline eda-modbus-bridge/CHANGELOG.md:13]: Change logs are normally "newest first", we can probably change the order the next time we make a change?

**Pull request merged**